airstack-modules/modules/airstack/common/index.ts

```diff
@@ -121,7 +157,63 @@ export function getOrCreateAirBlock(
     block.hash = blockHash;
     block.number = blockHeight;
     block.timestamp = blockTimestamp
-    block.save()
   }
   return block as AirBlock;
+}
+
+/**
+ * @dev this function does not save the returned entity
+ * @dev this function gets or creates a new air account entity
+ * @param chainId chain id
+ * @param address account address
+ * @param block air block object
+ * @returns AirAccount entity
+ */
+export function getOrCreateAirAccount(chainId: string, address: string, block: AirBlock): AirAccount {
+  let id = chainId.concat("-").concat(address);
+  let entity = AirAccount.load(id);
+  if (entity == null) {
+    entity = new AirAccount(id);
+    entity.address = address;
+    entity.createdAt = block.id;
```

Comment:
**Suggestion:** If we can create an empty object with only id. Then the caller can easily identify by comparing some of the fields, if its new object or old object. It will be very easy to write few logics on top of that.

The assigning of the values can be done by the caller.

This is just a suggestion, not expecting to change anything as of now here.

Reply:
ok